www/src/scss/_theme-light.scss

```diff
@@ -36,8 +36,7 @@ $alt-bg: #fafafa;
 $dialog-bg: #fff;
 $base-text: #4e4e4e;
 
-.light-theme,
-.custom-theme {
```

Comment:
bad merge? pretty sure this is needed.

Reply:
@tsingh777 This is fine, though you're right it looks wonky. It's just deleting `.custom-theme`